domains/mail.gh0stp4wz.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "GH0STP4WZ",
+    "email": "R3DGH0STP4WZ@protonmail.com"
+  },
+  "records": {
+    "A": "45.82.121.115"

```

A records is an Array
```suggestion
    "A": ["45.82.121.115"]
```